CMakeLists.txt

```diff
@@ -4,6 +4,7 @@
 
 cmake_minimum_required(VERSION 3.5)
 PROJECT(GazeboYARPPlugins)
+option(DISABLE_IMPLICIT_NETWORK_WRAPPERS "use new nws and nwc instead of old ones" OFF)
```

Comment:
The description of the option seems to be the old one (see discussion in https://github.com/robotology/gazebo-yarp-plugins/pull/565#discussion_r678091530) and the project-specific prefix is still missing?

Reply:
fixed